functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -0,0 +1,159 @@
+package __performance_profile_creator
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/ghodss/yaml"
+
+	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	"github.com/openshift-kni/performance-addon-operators/cmd/performance-profile-creator/cmd"
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+)
+
+const (
+	mustGatherPath       = "../../testdata/must-gather"
+	expectedProfilesPath = "../../testdata/ppc-expected-profiles"
+	ppcPath              = "../../build/_output/bin/performance-profile-creator"
+
+	argsDelimiter     = "_"
+	keyValueDelimiter = ":"
+)
+
+var _ = Describe("[rfe_id:xxxxx][ppc] Performance Profile Creator", func() {
```

Comment:
Adding a note here to make sure that we populate the rfe_id before we merge this PR.

Reply:
done